pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -270,24 +287,67 @@ def update_map_location(self):
             self.last_map_update = now
 
     def snipe(self, pokemon):
-        """Snipe a Pokemon by teleporting.
-
+        """
+		Snipe a Pokemon by teleporting.
         Args:
             pokemon: Pokemon to snipe.
         """
+
+        # Backup position before anything
         last_position = self.bot.position[0:2]
+
+        # Teleport, so that we can see nearby stuff
         self.bot.heartbeat()
         self._teleport_to(pokemon)
-        catch_worker = PokemonCatchWorker(pokemon, self.bot, self.config)
-        api_encounter_response = catch_worker.create_encounter_api_call()
-        time.sleep(self.config.get('snipe_sleep_sec', 2))
-        self._teleport_back(last_position)
-        self.bot.api.set_position(last_position[0], last_position[1], self.alt, False)
+
+        exists = False
+        nearby_pokemons = []
+        nearby_stuff = self.bot.get_meta_cell()
```

Comment:
Can you tell me why? I've been testing this for a whole day and results are impressive. Did you even test it?